docs/rackhd/event_notification.rst

```diff
@@ -244,33 +244,43 @@ The `payload.json` attributes in the example above are as below:
+filters   Array  **optional** An array of conditions that decides which events should be notified to hook url.
 ========= ====== ============ ============================================
 
-An example of `payload.json` with minimal attributes
+When a hook is registered and eligible events happened, RackHD will send a ``POST request`` to the hook url. POST request's ``Content-Type`` will be ``application/json``, and the request body be the event payload.
+
+An example of `payload.json` with minimal attributes:
 
 .. code-block:: JSON
 
     {
         "url": "http://www.abc.com/def"
     }
 
-RackHD will send a ``POST request`` to the hook url when a hook is registered, its  ``Content-Type`` is ``application/json``, and the event payload is in the request body.
-
-When multiple hooks are registered, each 'url' in the hooks will receive the events.
+When multiple hooks are registered, a single event will be sent to multiple hook urls if it meets filtering conditions.
 
 Event Filter Rules
 ^^^^^^^^^^^^^^^^^^^
 
-The conditions of which events are notified could be specified in the `filters` attribute in the hook_payload_, when `filters` attribute is not specified, or it's empty, all the events will be notified.
+The conditions of which events should be notified could be specified in the `filters` attribute in the hook_payload_, when `filters` attribute is not specified, or it's empty, all the events will be notified to the hook url.
+
+The `filters` attribute is an array, so multiple filters could be specified. The event will be sent as long as any filter condition is satisfied, even if the conditions may have overlaps.
 
-The `filters` attribute is an array, so multiple filters could be specified. The event will be sent as long as one filter condition is satisfied, even if the conditions may have overlaps.
+The filter attributes are `type`, `typeId`, `action`, `severity` and `nodeId` listed in event_payload_. Filtering by `data` is not supported currently. Filtering expression of hook `filters` is based on javascript regular expression, below table describes some base operations for hook filters:
 
-The filter attributes are `type`, `typeId`, `action`, `severity` and `nodeId` listed in event_payload_. When one attribute is not specified or its value is wildcard ``*``, it means any values in that attribute could meet conditions.
+=========== =============================================== ======================================================= ============================
+Operation   Description                                     Example                                                 Eligible Events
+=========== =============================================== ======================================================= ============================
+equal       Attribute equals some value                     {"action": "discovered"}                                Events with `action` equals `discovered`
```

Comment:
Actually the strict equal is `^discovered$` rather than `discovered`. `discovered` means if the action contains any sub string of `discovered`, it passes the validation, such as "my.discovered" and "discovered.abc"